### ISSUE TITLE
About setting of batch size and learning rate

### ISSUE THREAD

**Issue by mingliangzhang2018** [hyperconnect/LADE#2]
Excuse me, do you test the PC Softmax on the same experiment setting of LADE about batch size and learning rate? I find that PC Softmax even get higher accuracy in some datasets such as ImageNet-LT, Places-LT and iNaturalist 2018. 
![image](https://user-images.githubusercontent.com/38106776/110887069-9a8e2400-8324-11eb-8692-ec3f3708f404.png)
inital learning rate is 0.05

**Comment by juice500ml** [comment]
Yes, we used exactly the same settings. The detailed settings can be found below (screenshot from the paper). The table that you've shown us is the hyperparameters for the LADER loss (lambda, Eq 15) and LADE loss (alpha, Eq 20).
![image](https://user-images.githubusercontent.com/7028312/110913791-e8248400-8358-11eb-9c0e-844b1408b510.png)

**Comment by mingliangzhang2018** [comment]
Excuse me, I think PC Softmax is a really strong baseline. The results in your paper underestimate the performance of PC softmax. 

For example,  setting the same experiment setting of LADE（inital learning rate is 0.05，batch size is 256, training epcoh is 90, backbone is ResNeXt-50), PC softmax get accuracy of 52.2% much higher than paper 48.9%, even surpass the LDAE loss 51.9%.  Simailar phenomena also exist in other datasets.

 I am wondering about the superiority of LDAE loss which is much more complex than PC softmax.

**Comment by juice500ml** [comment]
First of all, we agree that PC Softmax is a strong baseline, and LADE improves PC Softmax just a bit more. We believe that one of our paper's contribution is to rediscover the power of PC Softmax, which were quite widely ignored in the comunity.

Also, we found out that with extensive hyperparameter search, LADE outperforms PC Softmax, but we agree that it can be somewhat expensive to do so. 

We also believe your experimental results are indeed possible, as we noticed that the experimental settings are sometimes prone to randomness. This is why we also tried on 180 epoch settings. Have you tried that setting also?

**Comment by mingliangzhang2018** [comment]
Thank you very much for your reply.  The performance gap between the two methods is puny, which could be even attributed to randomness by default.  But I think the idea of the paper is interesting and worth further exploring, even though the experiment results are not ideal。

**Comment by smallcube** [comment]
> Excuse me, I think PC Softmax is a really strong baseline. The results in your paper underestimate the performance of PC softmax.
> 
> For example, setting the same experiment setting of LADE（inital learning rate is 0.05，batch size is 256, training epcoh is 90, backbone is ResNeXt-50), PC softmax get accuracy of 52.2% much higher than paper 48.9%, even surpass the LDAE loss 51.9%. Simailar phenomena also exist in other datasets.
> 
> I am wondering about the superiority of LDAE loss which is much more complex than PC softmax.

Can you kindly share the specific pytorch version to get the 52.2% accuracy? I am using Pytorch 1.7 and train the PC-Softmax on one RTX 3090. The obtained accuracy is only 47% on testing set (50K images).

**Comment by mingliangzhang2018** [comment]
To my knowledge, the pytorch version has little impact. Please check bugs in the program and try again.
